Makefile

```diff
@@ -31,7 +32,7 @@ bin/%:
 
 # go get -u github.com/onsi/ginkgo/ginkgo
 test:
-	#ginkgo -v -r
+	ginkgo -r controllers/rbac
```

Comment:
`-v` was removed as the envTest integration tests are super noisy in verbose mode.